231016.121318 ERROR [ConsumerCoordinator] [Consumer clientId=consumer-mainGroup-1, groupId=mainGroup] Offset commit with offsets {multi-partition-0=OffsetAndMetadata{offset=1949, leaderEpoch=1, metadata=''}} failed
org.apache.kafka.clients.consumer.RetriableCommitFailedException: Offset commit failed with a retriable exception. You should retry committing the latest consumed offsets.
Caused by: org.apache.kafka.common.errors.NotCoordinatorException: This is not the correct coordinator.
231016.121409 ERROR [ProcessAlarm] ---------------_QUEUE_SIZE = 10000
231016.121409 ERROR [ProcessAlarm] ---------------_mAlarmQueue.size = 142
231016.121409 ERROR [ProcessAlarm] ---------------_queue2G.size = 0
231016.121409 ERROR [ProcessAlarm] ---------------_queue3G.size = 0
231016.121409 ERROR [ProcessAlarm] ---------------_queue4G.size = 0
231016.121409 ERROR [ProcessAlarm] ---------------_queueCore.size = 0
231016.121409 ERROR [ProcessAlarm] ---------------_Loi ProcessAlarm run(): java.lang.NullPointerException: Cannot invoke "java.sql.Connection.prepareStatement(String)" because "connection" is null
231016.122216 ERROR [ConsumerCoordinator] [Consumer clientId=consumer-mainGroup-1, groupId=mainGroup] Offset commit with offsets {multi-partition-0=OffsetAndMetadata{offset=5551, leaderEpoch=1, metadata=''}} failed
org.apache.kafka.clients.consumer.RetriableCommitFailedException: Offset commit failed with a retriable exception. You should retry committing the latest consumed offsets.
Caused by: org.apache.kafka.common.errors.DisconnectException
231016.122216 ERROR [ConsumerCoordinator] [Consumer clientId=consumer-mainGroup-1, groupId=mainGroup] Offset commit with offsets {multi-partition-0=OffsetAndMetadata{offset=5557, leaderEpoch=1, metadata=''}} failed
org.apache.kafka.clients.consumer.RetriableCommitFailedException: Offset commit failed with a retriable exception. You should retry committing the latest consumed offsets.
Caused by: org.apache.kafka.common.errors.DisconnectException
